### ISSUE TITLE
[docs] Issue with 'web hint development configuration'

### ISSUE THREAD

**Issue by whiplash696** [webhintio/hint#4260]
https://webhint.io/docs/user-guide/configurations/configuration-development/index.html

**Comment by antross** [comment]
Thanks for the report @whiplash696, can you share more detail on what issue(s) you found with the documentation on this page? Thanks!

**Comment by masi** [comment]
The list of rules included by the ruleset is incomplete. You have to dig into the source code to find out what you get when you extend from it. This contradicts "The following code snippet is an expanded version of the previous code snippet." and is disappointing.

**Comment by antross** [comment]
Good point @masi (and it looks like we'd need to do this for all the configuration docs). I could see fixing this by either including the full JSON or linking to the source on GitHub. What do you think? 

The link is definitely the simplest approach to get the current "right" answer, but takes you away from the rest of the docs.

Putting the full JSON inline (along with the current partial snippet) risks getting out-of-sync if we forget to update it. But we could add some build code to copy it from the actual JSON file to avoid that problem.

**Comment by masi** [comment]
I agree that copying manually large lists easily ends up being stale. As a quick fix linking to the source files is IMHO good enough.

**Comment by sarvaje** [comment]
Thanks @masi!

Would you be interested in opening a PR for this?

This should be the link to replace the expanded code snipped with: https://github.com/webhintio/hint/blob/main/packages/configuration-development/index.json

The markdown for this page lives here: https://github.com/webhintio/hint/blob/main/packages/configuration-development/README.md
